src/ui/pages/SidePage/components/WalletConnect/WalletConnectStageTwo.tsx

```diff
@@ -39,7 +39,7 @@ const WalletConnectStageTwo = ({
     useState<IdentifierShortDetails | null>(null);
 
   const displayIdentifiers = identifierCache
-    .filter((item) => !item.groupMetadata)
+    .filter((item) => !item.multisigManageAid)
```

Comment:
We have to filter for both `groupMetadata` and `multisigManageAid`

Reply:
I resolved your comment. please help me check it.